content/security/team.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: security
+title: "About the Jenkins Security Team"
+section: security
+---
+
+The Jenkins Security Team is a group of volunteers led by the link:https://wiki.jenkins.io/display/JENKINS/Team+Leads[Jenkins Security Officer].
+Our goal is to improve the security of Jenkins and to give administrators the tools and information they need to secure their Jenkins instances.
+
+
+## What We Do
+
+. We look for security issues in Jenkins and plugins
+. We process incoming reports of security issues in any component
+. We work with reporters and maintainers to get security issues fixed
+. We develop Jenkins (core) security fixes
+. We review and test security fixes in core and plugins
+. We write and publish security advisories
+. We implement security-related features and improvements (link:#contribute[and you can too!])
+
+Security vulnerabilities in core are typically resolved by members of this team, while vulnerabilities in plugins are generally resolved in collaboration with the plugin maintainer(s).
+
+
+[[join]]
+## Joining the Team
+
+Members of the security team have access to sensitive information, so membership is limited to people with a history of contributions to the Jenkins project and is subject to approval by the Jenkins Security Officer.
+
+Contact the Jenkins security team via email to the private `jenkinsci-cert@googlegroups.com` mailing list if you're interested.
+// TODO Does it make sense to keep this on the dev list? For visibility probably? OTOH, people might not contact us due to the potential for public rejection.
+
+### Expectations
+
+As a member of the security team you will be expected to regularly contribute, or you may be removed from the team again to limit the exposure of sensitive security-related information.
+Security team membership is specifically for those working on Jenkins (core) security fixes or on improving Jenkins security beyond specific components.
+These contributions need to be not otherwise feasible without the additional access to sensitive information granted through the membership.
+For example, as a plugin maintainer, you are already able to address security issues in plugins you maintain even without being a security team member.
+
+There are many ways to contribute to the security of Jenkins that don't require you to be on the security team.
+Learn more: link:#contribute[Other Ways to Contribute]
+
+
+### Technical Requirements
+
+In order to join, you'll need to have:
+
+. a link:https://accounts.jenkins.io/[Jenkins community account] and have logged in to Artifactory and Jira before,
+. a link:https://github.com[GitHub] account with link:https://help.github.com/articles/securing-your-account-with-two-factor-authentication-2fa/[two-factor authentication enabled],
+. a link:https://github.com/jenkinsci/infra-cla/[CLA] in place, and
```

Comment:
Expand the acronym for those us that don't know what it means?
```suggestion
. a link:https://github.com/jenkinsci/infra-cla/[Contributor License Agreement (CLA)] in place, and
```

Reply:
Good point. Was about to bring up that readers only need to click the link, but not even the readme of the linked repo explains it, it's only in the repo description.